### PR TITLE
fix files ignoring in secrets command for packs

### PR DESCRIPTION
**Pull request by esharf**
## Status
Ready.

## Related Issues
fixes: [#221179](https://github.com/demisto/etc/issues/21179)

## Description
fix ignoring file on packs white list.

### CODE REVIEW

File: demisto_sdk/validation/secrets.py [hunk 1]
```diff
@@ -332,10 +332,17 @@ def get_generic_white_list(whitelist_path):
...
         final_white_list = []
+        files_white_list = []
+
         if os.path.isfile(whitelist_path):
             with io.open(whitelist_path, mode="r", encoding="utf-8") as secrets_white_list_file:
-                final_white_list = secrets_white_list_file.read().split('\n')
-        return final_white_list, [], []
+                temp_white_list = secrets_white_list_file.read().split('\n')
+            for i in range(len(temp_white_list)):
+                if '/' in temp_white_list[i]:
```
Comment (ronykoz) [inline demisto_sdk/validation/secrets.py]: doesn't seem like a good identifier for files, try to think of something else.
Like os.path.exists

File: demisto_sdk/main.py [hunk 2]
```diff
@@ -350,7 +350,8 @@ def run(**kwargs):
 # ====================== run-playbook ====================== #
 @main.command(name="run-playbook",
               short_help="Run a playbook in Demisto. "
-                         "DEMISTO_API_KEY environment variable should contain a valid Demisto API Key. "
+                         "DEMISTO_API_KEY environment variable "
```
Comment (yaakovi) [inline demisto_sdk/main.py]: why you broke the line? please revert this change

File: demisto_sdk/validation/secrets.py [hunk 3]
```diff
@@ -330,12 +332,25 @@ def get_generic_white_list(whitelist_path):
...
+    def get_packs_white_list(whitelist_path, pack_name=None):
         final_white_list = []
+        files_white_list = []
+
         if os.path.isfile(whitelist_path):
             with io.open(whitelist_path, mode="r", encoding="utf-8") as secrets_white_list_file:
-                final_white_list = secrets_white_list_file.read().split('\n')
-        return final_white_list, [], []
+                temp_white_list = secrets_white_list_file.read().split('\n')
+            for i in range(len(temp_white_list)):
```
Comment (yaakovi) [inline demisto_sdk/validation/secrets.py]: this is C-code.
please use `for line in temp_white_list`
